hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/SimpleConfiguration.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Example configuration to test the configuration injection.
  */
-@ConfigGroup(prefix = "ozone.scm.client")
+@ConfigGroup(prefix = "fake.scm.client")
```

Comment:
can we name the prefix like test.scm.client?

Reply:
Good idea, thanks.